content/techniques/cookies.md

```diff
@@ -49,7 +49,7 @@ findAll(@Res({ passthrough: true }) response: Response) {
 }
 ```
 
-> warning **Attention** Si vous voulez laisser la logique de gestion des réponses au framework, n'oubliez pas de mettre l'option `passthrough` à `true`, comme indiqué ci-dessus. Pour en savoir plus, cliquez ici (/controllers#approche-spécifique-aux-bibliothèques).
+> warning **Attention** Si vous voulez laisser la logique de gestion des réponses au framework, n'oubliez pas de mettre l'option `passthrough` à `true`, comme indiqué ci-dessus. Pour en savoir plus, cliquez [ici](/controllers#appendix-library-specific-approach).
```

Comment:
attention à retraduire l'ancre correctement, sinon le lien ne fonctionnera pas

Reply:
I have reverted the link as it was initially